theatre/studio/src/panels/SequenceEditorPanel/RightOverlay/FocusRangeZone/FocusRangeThumb.tsx

```diff
@@ -112,25 +112,32 @@ const ColoredMargin = styled.div<{type: 'start' | 'end'; enabled: boolean}>`
   pointer-events: none;
 
   ${() => RangeStrip}.dragging ~ ${TheDiv} > & {
-    background: ${focusRangeStripTheme.dragging.backgroundColor};
+    --bg: ${focusRangeStripTheme.dragging.backgroundColor};
   }
 
-  background-color: ${({enabled}) =>
+  --bg: ${({enabled}) =>
     enabled
       ? focusRangeStripTheme.enabled.backgroundColor
       : focusRangeStripTheme.disabled.backgroundColor};
 
   // highlight the handle if it's hovered, or the whole strip is hovverd
   ${() => RangeStrip}:hover ~ ${TheDiv} > & {
-    background: ${focusRangeStripTheme.hover.backgroundColor};
-    stroke: ${focusRangeStripTheme.hover.stroke};
+    --bg: ${focusRangeStripTheme.hover.backgroundColor};
   }
 
-  width: 8px;
+  background: linear-gradient(
```

Comment:
This, along with the [previous](1e243b8c8e76303e646e6b44c809ddda9a51d857) and [next](42d8dc930751191c9f2f483fa4a0a4ee49d40653) commits fix the following visual glitch:

The thumbs were interfering with the frame grid:

<img width="829" alt="image" src="https://user-images.githubusercontent.com/593118/166229370-295a2508-4c36-4764-93e0-e9b87c1b3a45.png">

https://user-images.githubusercontent.com/593118/166229553-f8bc528b-ff5f-454a-b8a8-d7a127f59319.mp4

After:


https://user-images.githubusercontent.com/593118/166230110-d0493e7f-53ea-48d9-8a61-b7af36f51042.mp4